infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -292,16 +283,14 @@ private void initializationTask() {
                     continue;
                 }
                 // If recovery succeeds, reconfiguration was successful.
-                triggerSequencerBootstrap(serverContext.copyManagementLayout());
-                recoveryBarrierFuture.complete(true);
+                // Save the latest management layout.
+                serverContext.saveManagementLayout(getCorfuRuntime().getLayoutView().getLayout());
+
                 log.info("Recovery completed");
             }
 
-            // Sequencer bootstrap required if this is fresh startup.
-            if (!recoveryBarrierFuture.isDone()) {
-                bootstrapPrimarySequencerServer();
```

Comment:
We can remove this method now as suggested by sonarqube

Reply:
Done